go/api/response_types.go

```diff
@@ -7,6 +7,11 @@ type Result[T any] struct {
 	isNil bool
 }
 
+type KeyWithMemberAndScore struct {
+	Key, Member string
```

Comment:
```suggestion
	Key string
	Member string
```
Please add documentation, similar to type ServerCredentials struct {}